fil-proofs-tooling/build.rs

```diff
@@ -0,0 +1,8 @@
+use std::error::Error;
+use vergen::EmitBuilder;
+
+fn main() -> Result<(), Box<dyn Error>> {
+    // Emit the instructions
```

Comment:
Could you please change this comment to something like

    // Emits the `VERGEN_GIT_SHA` and `VERGEN_GIT_COMMIT_TIMESTAMP` environment variables.

This way if someone searches where those come from, they'll find this comment.